src/app/api/rulebookApi.ts

```diff
@@ -62,6 +67,29 @@ const getEventLog = (): Promise<EventLogResponse> =>
 const getActionLog = (): Promise<ActionLogResponse> =>
   apiClient<ActionLogResponse>('action-log').then((response) => response.data);
 
+const getInventory = (): Promise<InventoryResponse> =>
+  apiClient<InventoryResponse>('inventory').then((response) => response.data);
+
+const saveInventory = (inventory: string): Promise<Response<InventoryResponse>> => {
+  return apiClient.post<InventoryResponse>('inventory', { inventory });
+};
+
+const getRulebookStatus = (): Promise<{ enabled: boolean }> => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve({ enabled: MOCKED_RULEBOOK_STATUS });
+    }, 100);
+  });
+};
```

Comment:
This is necessary in order to use a switch component in the UI. The current status should be fetched from the API too.